### ISSUE TITLE
OpenShift Java Console fails to open with v3.5.34 or higher

### ISSUE THREAD

**Issue by tadayosi** [fabric8io/fabric8-maven-plugin#1268]
### Description
OpenShift Java Console fails to open when you `mvn fabric8:deploy` the following quickstart (`master` branch), which uses version 3.5.34 of fabric8-maven-plugin:
https://github.com/fabric8-quickstarts/spring-boot-camel-xml
However, when you downgrade the plugin to 3.5.33 the console works fine.

Is it a regression, possibly related to https://issues.jboss.org/browse/OSFUSE-700 ?

### Info

* f-m-p version : >= 3.5.34
* Maven version (`mvn -v`) : 
```
Apache Maven 3.5.3 (3383c37e1f9e9b3bc3df5050c29c8aff9f295297; 2018-02-25T04:49:05+09:00)
```
* Kubernetes / OpenShift setup and version : 
```
$ oc version
oc v3.9.0+191fece
kubernetes v1.9.1+a0ce1bc657
features: Basic-Auth

Server https://localhost:8443
openshift v3.9.0+191fece
kubernetes v1.9.1+a0ce1bc657
```
* If it's a bug, how to reproduce : See above

**Comment by rohanKanojia** [comment]
Which console are you talking about? Is it the Openshift web console, that you can access by doing `minishift console`(if using minishift)?

**Comment by tadayosi** [comment]
@rohanKanojia I've been talking about "Java Console", which you can find the link at the pod page of a Jolokia-enabled Java app. I'm using `oc cluster up`.

![screen shot 2018-04-12 at 10 31 40](https://user-images.githubusercontent.com/156692/38651528-bc2a2c42-3e3c-11e8-90f3-1482caae8c79.png)

**Comment by tadayosi** [comment]
FWIW, `Dockerfile` generated by fabric8-maven-plugin 3.5.34 uses:

    FROM fabric8/s2i-java:2.1

whereas `Dockerfile` generated by the working plugin version 3.5.33 uses:

    FROM fabric8/s2i-java:2.0

This seems to be the difference which is causing this issue.

**Comment by rhuss** [comment]
The Java console is provided when Jolokia is enabeled in the base image at port 8778.

@tadayosi what exact error did you get when connecting to the Java console ?

I'm going to reproduce this with a local installation.

**Comment by rhuss** [comment]
I can confirm the issue. The reason seems that in the base image used in 3.5.34 in `jolokia.properties` --> `protocol=https` is missing:

3.5.34 properties
```
host=*
port=8778
discoveryEnabled=false
user=jolokia
password=x6fXoCMrgiJwa1Ix4XcBpQnTvQKm2f
useSslClientAuthentication=true
extraClientCheck=true
caCert=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
clientPrincipal=cn=system:master-proxy
```

3.5.33 properties
```
host=*
port=8778
discoveryEnabled=false
user=jolokia
password=whqN3jJyNCXtWaGIJzDxfndUqqrqoU
useSslClientAuthentication=true
extraClientCheck=true
protocol=https
caCert=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
clientPrincipal=cn=system:master-proxy
```

going to find out now, what changed in the base images ...

**Comment by rhuss** [comment]
Ok, solution is simple --> #1336

Actually it was a regression in the base images which has been fixed around February this year, but f-m-p hasn't been updated to use the fixes https://github.com/fabric8io-images/s2i image.

**Comment by rohanKanojia** [comment]
Closed via #1336
